sdk/eidolon_ai_sdk/apu/longterm_memory_unit.py

```diff
@@ -0,0 +1,146 @@
+from enum import Enum
+from typing import Optional, List, Callable
+
+from pydantic import BaseModel
+from qdrant_client.http.models import ScoredPoint
+
+from eidolon_ai_sdk.agent_os import AgentOS
+from eidolon_ai_sdk.apu.call_context import CallContext
+from eidolon_ai_sdk.apu.llm_message import LLMMessage
+from eidolon_ai_sdk.apu.llm_unit import LLMUnit
+from eidolon_ai_sdk.apu.mem0 import EidolonMem0
+from eidolon_ai_sdk.apu.processing_unit import ProcessingUnit
+from eidolon_ai_sdk.system.reference_model import Reference
+from eidolon_ai_sdk.system.reference_model import Specable
+
+
+class LongTermMemoryUnitScope(Enum):
+    SYSTEM = "system"
+    AGENT = "agent"
+    USER = "user"
+    USER_AGENT = "userAgent"
+    USER_PROCESS = "userProcess"
```

Comment:
You mentioned that you were going to change this? presumably by turning unit_scope into List[LongTermMemoryUnitScope] and allowing them to be used in conjunction? then restring the options to be user, agent, and process

Reply:
i'll make that change